car_plate.go

```diff
@@ -0,0 +1,31 @@
+package faker
```

Comment:
Please move this to the https://github.com/jaswdr/faker/blob/master/car.go file, creating a `Plate()` method that return a fake car plate.